sycl/test-e2e/Matrix/SG32/joint_matrix_tf32.cpp

```diff
@@ -5,17 +5,14 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix-tf32
+// REQUIRES: aspect-ext_intel_matrix
```

Comment:
matrix-tf32 elimination looks good because I see we are using type-based query in the hpp file

Reply:
yes, I added usage of runtime query in all cases, where aspect is not enough and where it doesn't make sense to use UNSUPPORTED tag.